doc/cabal-commands.rst

```diff
@@ -403,3 +403,63 @@ the custom publishing of Haddock documentation to Hackage.
   While this is happening, we recommend not uploading these packages
   to Hackage (and instead referencing the package directly
   as a ``source-repository-package``).
+
+cabal gen-bounds
+----------------
+
+``cabal gen-bounds [FLAGS]`` generates bounds for all dependencies that do not
+currently have them.  Generated bounds are printed to stdout. You can then
+paste them into your .cabal file.
+
+See `the section on generating dependency version bounds <cabal-package.html#generating-dependency-version-bounds>`__ for more details and examples.
+
+cabal outdated
+--------------
+
+``cabal outdated [FLAGS]`` checks for outdated dependencies in the package
+description file or freeze file.
+
+``cabal outdated`` supports the following flags:
+
+- ``--freeze-file``: Read dependency version bounds from the freeze file
```

Comment:
Perhaps call it `--v1-freeze-file` and don't mention any alias? Presumably we will soon make `--freeze-file` and alias for `--v2-freeze-file` so better start preparing users for that.

Reply:
`--v1-freeze-file` does definitely not exist.